test/framework/robot.py

```diff
@@ -713,6 +713,47 @@ def test_search_paths(self):
         regex = re.compile(r"^ \* %s$" % os.path.join(self.test_prefix, test_ec), re.M)
         self.assertTrue(regex.search(outtxt), "Found pattern %s in %s" % (regex.pattern, outtxt))
 
+    def test_github_det_easyconfig_paths_from_commit(self):
+        """Test det_easyconfig_paths function in combination with --from-commit."""
+
+        test_ecs_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'easyconfigs', 'test_ecs')
+
+        commit = '589282cf52609067616fc2a522f8e4b81f809cb7'
+        args = [
+            os.path.join(test_ecs_path, 't', 'toy', 'toy-0.0.eb'),  # absolute path
+            'toy-0.0-iter.eb',  # relative path, available via robot search path
+            # commit in which ReFrame-4.3.2.eb was added,
+            # see https://github.com/easybuilders/easybuild-easyconfigs/pull/18763/commits
+            '--from-commit', commit,
+            'ReFrame-4.3.2.eb',  # easyconfig included in commit, should be resolved via robot search path
+            '--dry-run',
+            '--robot',
+            '--robot=%s' % test_ecs_path,
+            '--unittest-file=%s' % self.logfile,
+            '--github-user=%s' % GITHUB_TEST_ACCOUNT,  # a GitHub token should be available for this user
```

Comment:
The comment here is confusing me, as this test looks to be able to be run without a github token.

If a token is required then add the
```py
        if self.github_token is None:
            print("Skipping test_from_pr, no GitHub token available?")
            return
```
or amend the comment if the token is not required.

Reply:
fixed in [5f169a9](https://github.com/easybuilders/easybuild-framework/pull/4468/commits/5f169a9f7845cef045a2365f9a25cbd452cb40fe)